.github/workflows/sync-issue-labels-compute.yml

```
@@ -0,0 +1,207 @@
+          egress-policy: audit
+
+      - name: Compute linked issue labels
+        id: compute
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          PR_NUMBER: ${{ github.event.pull_request.number || github.event.inputs.pr_number }}
+          DRY_RUN: 'true'
+          REQUESTED_DRY_RUN: ${{ github.event_name == 'workflow_dispatch' && format('{0}', github.event.inputs['dry-run-enabled']) || 'true' }}
+          IS_FORK_PR: ${{ github.event_name == 'pull_request' && github.event.pull_request.head.repo.fork || 'false' }}
+          MAX_LINKED_ISSUES: '20'
+        uses: actions/github-script@ed597411d8f924073f98dfc5c65a23a2325f34cd # v8.0.0
```

YAMLlint (1.38.0)

[warning] too few spaces before comment: expected 2

(comments)